src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -91,6 +96,22 @@ export const Accept: FC<AcceptProps & StripeProps> = props => {
       const orderOrError = (await acceptOffer()).commerceBuyerAcceptOffer
         ?.orderOrError
 
+      if (orderOrError?.actionData?.clientSecret) {
+        const scaResult = await stripe.handleCardAction(
+          orderOrError.actionData.clientSecret
+        )
+
+        if (scaResult.error) {
```

Comment:
While this isn't a dead-end, I'd like to track this event to get a better idea of how often users see these error dialogs. Could you please include this blurb before showing the error dialog?

```ts
trackEvent({
  action: ActionType.errorMessageViewed,
  context_owner_type: OwnerType.ordersAccept,
  context_owner_id: props.order.internalID,
  title: "An error occurred",
  message: scaResult.error.message,
  error_code: code || null,
  flow: "user accepts offer",
})
```

Reply:
Sure, https://github.com/artsy/force/pull/12868/commits/5115dc6a6ab919128a18a999d8e8cac73f10009b